sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -126,7 +125,11 @@ dif_rstmgr_result_t dif_rstmgr_reset_lock(const dif_rstmgr_t *handle,
   }
 
   mmio_region_t base_addr = handle->params.base_addr;
-  rstmgr_software_reset_lock(base_addr, peripheral);
+  bitfield_bit32_index_t index = rstmgr_regen_rstctrl_bit_index(peripheral);
+  uint32_t bitfield = bitfield_bit32_write(UINT32_MAX, index, false);
+
+  // Clear bit to disable the software reset for the peripheral.
```

Comment:
nit: Should this comment move above for the sake of clarity?

Reply:
Yup, done.